packages/client/modules/userDashboard/components/OrgBilling/UpdatePayment.tsx

```diff
@@ -0,0 +1,97 @@
+import styled from '@emotion/styled'
+import {CardCvcElement, CardExpiryElement, CardNumberElement} from '@stripe/react-stripe-js'
+import React from 'react'
+import PrimaryButton from '../../../../components/PrimaryButton'
+import SecondaryButton from '../../../../components/SecondaryButton'
+import {PALETTE} from '../../../../styles/paletteV3'
+
+const UpgradeButton = styled(PrimaryButton)<{isDisabled: boolean}>(({isDisabled}) => ({
+  background: isDisabled ? PALETTE.SLATE_200 : PALETTE.SKY_500,
+  color: isDisabled ? PALETTE.SLATE_600 : PALETTE.WHITE,
+  boxShadow: 'none',
+  marginTop: 16,
+  width: '100%',
+  elevation: 0,
+  '&:hover, &:focus': {
+    boxShadow: 'none',
+    background: isDisabled ? PALETTE.SLATE_200 : PALETTE.SKY_600
+  }
+}))
+
+const CARD_ELEMENT_OPTIONS = {
+  style: {
+    base: {
+      color: PALETTE.SLATE_800,
+      fontFamily: '"IBM Plex Sans", sans-serif',
+      fontSmoothing: 'antialiased',
+      fontSize: '16px',
+      '::placeholder': {
+        color: PALETTE.SLATE_600
+      }
+    }
+  }
+}
+
+type Props = {
+  setIsUpdating: (isUpdating: boolean) => void
```

Comment:
-1 If it's not temporary, then we shouldn't pass a state setter of a parent in here directly. Instead we should have an `onCancel` handler (and an `onSubmit` if that shouldn't be a noop)